config/docker/cros-tast/ssh_retry.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+let tries=10
+
+if [ -z "$*" ] ; then
+    echo "Need to specify ssh options"
+    exit 1
+fi
+
+while [[ $tries -ne 0 ]] ; do
+    ssh $*
+    retcode=$?
+    if [[ $retcode -eq 0 ]] ; then
+        let tries=1
```

Comment:
I'd consider using `break` here. It looks more clear to me.

Reply:
Done! Thanks, no idea why i did like that, probably forgot that bash has break too